keyboards/redox/keymaps/tehl/keymap.c

```diff
@@ -0,0 +1,94 @@
+#include QMK_KEYBOARD_H
```

Comment:
Actually, it may be better to move this to the keymap.h file.  Then you shouldn't need the quantum.h include, either.

Reply:
As above